src/System.IO/src/project.json

```diff
@@ -9,6 +9,15 @@
         "dotnet5.7"
       ]
     },
+    "netstandard1.7": {
+      "dependencies": {
+        "Microsoft.NETCore.Platforms": "1.0.1",
```

Comment:
This looks like copy/paste but as an FYI I don't believe this package reference is needed for anyone that only depends on the private coreclr targeting pack package.